src/frontend/src/flows/faq/index.ts

```diff
@@ -8,69 +9,91 @@ export { questions } from "./questions";
 
 // The rendered (list item) question
 function renderQuestion(faq: Question) {
-  return html`<li id=${faq.anchor}>
-    <h3>${faq.question}</h3>
-    <p>${faq.answer}</p>
-    <ul class="links-list">
-      ${Object.values(faq.links).map(
-        (link) =>
-          html`<li>
-            &middot;
-            <a
-              class="textLink"
-              rel="noopener noreferrer"
-              href="${link.link}"
-              target="_blank"
-              >${link.name}</a
-            >
-          </li>`
-      )}
-    </ul>
+  return html`<li
+    class="faq__question"
+  >
+    <details
+    id=${faq.anchor} >
+    <summary class="faq__question-summary">
+      ${faq.question}
+    <div class="faq__question-underline"></div>
+    </summary>
+    <div>
+      <p class="faq__answer">${faq.answer}</p>
+      <ul class="faq__answer-links">
+        ${Object.values(faq.links).map(
+          (link) =>
+            html`<li>
+              &middot;
+              <a
+                class="faq__answer-link"
+                rel="noopener noreferrer"
+                href="${link.link}"
+                target="_blank"
+                >${link.name} &#8599;</a
+              >
+            </li>`
+        )}
+      </ul>
+    </div>
   </li>`;
 }
 
 // The FAQ page
 const pageContent = html`
   <style>
+    html,
+    body,
+    main {
+      height: max-content;
+      overflow-x: hidden;
+    }
+
+    body {
+      position: relative;
+    }
     /* briefly flash the question when redirected to a particular question */
     @keyframes flash-question {
       0% {
         background-color: transparent;
       }
       50% {
-        background-color: var(--grey-100);
+        background-color: var(--rainbow-orange);
+        border-radius: 0.3em;
       }
       100% {
         background-color: transparent;
       }
     }
-
-    a {
-      color: var(--grey-800);
-    }
-
-    ul {
-      list-style-type: none;
-    }
-
-    li {
-      border-radius: 0.5rem;
-      padding: 0.5rem;
-    }
     :target {
       animation-name: flash-question;
-      animation-duration: 3s;
+      animation-duration: 600ms;
     }
   </style>
-  <div class="container" id="faq">
-    <h1>FAQ</h1>
-    <ul>
-      ${Object.values(questions).map((faq) => renderQuestion(faq))}
-    </ul>
+  <div class="faq__container">
+    <div>
+      <h1 class="faq__title">FAQ</h1>
+      <div class="faq__title-underline"></div>
+      <ul class="faq__questions">
+        ${Object.values(questions).map((faq) => renderQuestion(faq))}
```

Comment:
I'd rather not rely on the ordering of `Object.values` here. I think we care about the order in which questions are displayed here, so we can put the most important ones at the top, right?

In that case we should probably use an Array here. Was the reason for using an object to make the access type safe? I think we can find other ways of achieving that if so.

Reply:
Done! Added priority and then sorting alphabetically.